build-support/docker/travis_ci_py27_ucs2/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+# This file duplicates travis_ci/Dockerfile, except it installs Python 2.7 instead of Python 3.6
```

Comment:
Why not have a single image with both pythons?

Reply:
I expanded the comment to explain this. tl;dr for why we don't have one single image: we don't want multiple Python 2 versions for any shard except for the Linux UCS shard, as that would make them all much more finicky in how they resolve the bin name `python2.7`.